extensions/nodes/node_status.go

```diff
@@ -116,11 +117,11 @@ func AllMachineReady(client *rancher.Client, clusterID string, timeout time.Dura
 	return err
 }
 
-// AllNodeDeleted is a helper method that will loop and check if the node is deleted in the cluster.
+// AllNodeDeleted is a helper method that will loop and check if all of the node is deleted in the cluster.
 func AllNodeDeleted(client *rancher.Client, ClusterID string) error {
 	ctx := context.Background()
 	err := wait.PollUntilContextTimeout(
-		ctx, oneSecondInterval, fiveMinuteTimeout, true, func(ctx context.Context) (bool, error) {
+		ctx, oneSecondInterval, defaults.ThirtyMinuteTimeout, true, func(ctx context.Context) (bool, error) {
```

Comment:
I think you found this one again by observation but at a glance 30 mins looks long to confirm a node deletion. Could you maybe put a comment in the code as to why its this long?

Reply:
Copy paste again. This one should be fine at 10 minutes, will test tomorrow to make sure.

Reply:
Resolved